src/css/sort_option.css

```diff
@@ -0,0 +1,120 @@
+.search_sort {
+  position: relative;
+  user-select: none;
+  width: 26rem;
+  font-size: 1rem;
+  font-weight: 300;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  height: 60px;
+  white-space: nowrap;
+}
+.search_sortTrigger {
+  position: relative;
+  display: grid;
+  height: 100%;
+  align-items: center;
+  padding: 0 22px;
+  color: var(--main-text-color);
+  background-color: var(--main-border-color);
+  cursor: pointer;
+}
+.search_sortTrigger::before {
```

Comment:
Moved the ⇵ icon to a pseudo-element since it's just for styling purposes.

I changed the font size to 1.5 rem from 24 px because 16px (default) * 1.5 = 24px;